From: "Y.C. Chen" <yc_chen@aspeedtech.com>
To: dri-devel@lists.freedesktop.org
Cc: yc_chen@aspeedtech.com, kuohsiang_chou@aspeedtech.com, eich@suse.com,
        airlied@redhat.com
Subject: [PATCH 2/2] drm/ast: AST2000 cannot be detected correctly
Date: Wed, 10 Sep 2014 12:07:54 +0800

From: "Y.C. Chen" <yc_chen@aspeedtech.com>
References: bnc#895983
Patch-mainline: submitted for upstream
Signed-off-by: Egbert Eich <eich@suse.de>

Type error and cause AST2000 cannot be detected correctly

Signed-off-by: Y.C. Chen <yc_chen@aspeedtech.com>
Reviewed-by: Egbert Eich <eich@suse.de>
---
 drivers/gpu/drm/ast/ast_main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/ast/ast_main.c b/drivers/gpu/drm/ast/ast_main.c
index b027e33..b792194 100644
--- a/drivers/gpu/drm/ast/ast_main.c
+++ b/drivers/gpu/drm/ast/ast_main.c
@@ -105,7 +105,7 @@ static int ast_detect_chip(struct drm_device *dev)
 			}
 			ast->vga2_clone = false;
 		} else {
-			ast->chip = 2000;
+			ast->chip = AST2000;
 			DRM_INFO("AST 2000 detected\n");
 		}
 	}
-- 
1.8.3.1


